Deployment is failing that it expects 2 noobaa endpoints, but only one exists

Job:
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/13233//consoleFull

```
12:59:42 - MainThread - ocs_ci.ocs.ocp - INFO - status of  at column STATUS - item(s) were ['Running'], but we were waiting for all 2 of them to be Running
12:59:42 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
12:59:45 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jenkins/current-cluster-dir/openshift-cluster-dir/auth/kubeconfig get Pod  -n openshift-storage --selector=noobaa-s3=noobaa -o yaml
12:59:45 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jenkins/current-cluster-dir/openshift-cluster-dir/auth/kubeconfig get Pod noobaa-endpoint-7646c966b4-q92mc -n openshift-storage
12:59:45 - MainThread - ocs_ci.ocs.ocp - INFO - status of  at column STATUS - item(s) were ['Running'], but we were waiting for all 2 of them to be Running
12:59:45 - MainThread - ocs_ci.ocs.ocp - ERROR - timeout expired: Timed Out: (600,)
12:59:45 - MainThread - ocs_ci.ocs.ocp - ERROR - Wait for Pod resource  at column STATUS to reach desired condition Running failed, last actual status was ['Running']
12:59:45 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jenkins/current-cluster-dir/openshift-cluster-dir/auth/kubeconfig describe Pod  --selector=noobaa-s3=noobaa

....
tests/ecosystem/deployment/test_deployment.py:25: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
ocs_ci/ocs/resources/storage_cluster.py:185: in ocs_install_verification
    timeout=timeout
ocs_ci/ocs/ocp.py:595: in wait_for_resource
    raise(ex)
ocs_ci/ocs/ocp.py:504: in wait_for_resource
    timeout, sleep, self.get, resource_name, True, selector
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <ocs_ci.utility.utils.TimeoutSampler object at 0x7fad989f5668>

    def __iter__(self):
        if self.start_time is None:
            self.start_time = time.time()
        while True:
            self.last_sample_time = time.time()
            try:
                yield self.func(*self.func_args, **self.func_kwargs)
            except Exception as ex:
                msg = f"Exception raised during iteration: {ex}"
                logging.error(msg)
            if self.timeout < (time.time() - self.start_time):
>               raise self.timeout_exc_cls(*self.timeout_exc_args)
E               ocs_ci.ocs.exceptions.TimeoutExpiredError: Timed Out: (600,)

ocs_ci/utility/utils.py:796: TimeoutExpiredError
---------------------------- Captured stdout setup -----------------------------```

--- a/conf/ocsci/noobaa_endpoints_scale.yaml
+++ b/conf/ocsci/noobaa_endpoints_scale.yaml
@@ -1,5 +1,0 @@
----
-#This config file specifies that the max noobaa endpoints will be scaled up to 5 instead of 1
-DEPLOYMENT:
-  min_noobaa_endpoints: 2
-  max_noobaa_endpoints: 5

--- a/ocs_ci/framework/conf/default_config.yaml
+++ b/ocs_ci/framework/conf/default_config.yaml
@@ -72,9 +72,6 @@ DEPLOYMENT:
   preserve_bootstrap_node: False
   # Terraform version
   terraform_version: "0.12.26"
-  # Following options are defaults for the min and max Noobaa endpoints
-  min_noobaa_endpoints: 2
-  max_noobaa_endpoints: 3
   # Following options are defaults for deployment with infra worker nodes
   infra_nodes: False
   # How long should ocs-ci wait for `openshift-install create cluster` run?

--- a/ocs_ci/framework/testlib.py
+++ b/ocs_ci/framework/testlib.py
@@ -1,7 +1,7 @@
 import pytest
 
 from ocs_ci.framework.pytest_customization.marks import *  # noqa: F403
-from ocs_ci.framework import config
+from ocs_ci.ocs.constants import MIN_NB_ENDPOINT_COUNT, MAX_NB_ENDPOINT_COUNT
 
 
 @pytest.mark.usefixtures(  # noqa: F405
@@ -44,5 +44,5 @@ class EcosystemTest(BaseTest):
     "nb_ensure_endpoint_count"
 )
 class MCGTest(ManageTest):
-    MIN_ENDPOINT_COUNT = config.DEPLOYMENT.get('min_noobaa_endpoints')
-    MAX_ENDPOINT_COUNT = config.DEPLOYMENT.get('max_noobaa_endpoints')
+    MIN_ENDPOINT_COUNT = MIN_NB_ENDPOINT_COUNT
+    MAX_ENDPOINT_COUNT = MAX_NB_ENDPOINT_COUNT

--- a/ocs_ci/ocs/constants.py
+++ b/ocs_ci/ocs/constants.py
@@ -1188,3 +1188,7 @@ SQUADS = {
 }
 
 PRODUCTION_JOBS_PREFIX = ['jnk']
+
+# min and max Noobaa endpoints
+MIN_NB_ENDPOINT_COUNT = 1
+MAX_NB_ENDPOINT_COUNT = 2

--- a/ocs_ci/ocs/resources/storage_cluster.py
+++ b/ocs_ci/ocs/resources/storage_cluster.py
@@ -8,7 +8,7 @@ from jsonschema import validate
 from ocs_ci.framework import config
 from ocs_ci.ocs import constants, defaults, ocp
 from ocs_ci.ocs.exceptions import ResourceNotFoundError
-from ocs_ci.ocs.node import get_compute_node_names, check_nodes_specs
+from ocs_ci.ocs.node import get_compute_node_names
 from ocs_ci.ocs.ocp import get_images, OCP
 from ocs_ci.ocs.resources.ocs import get_ocs_csv
 from ocs_ci.ocs.resources.pod import get_pods_having_label
@@ -139,13 +139,8 @@ def ocs_install_verification(
     ) == 4.5 and config.ENV_DATA.get('platform') == constants.AZURE_PLATFORM and post_upgrade_verification:
         rgw_count = 1
 
-    # Fetch the min and max Noobaa endpoints from the run config
-    if check_nodes_specs(min_cpu=constants.MIN_NODE_CPU, min_memory=constants.MIN_NODE_MEMORY):
-        min_eps = config.DEPLOYMENT.get('min_noobaa_endpoints')
-        max_eps = config.DEPLOYMENT.get('max_noobaa_endpoints')
-    else:
-        min_eps = 1
-        max_eps = 1 if float(config.ENV_DATA['ocs_version']) < 4.6 else 2
+    min_eps = 1
+    max_eps = 2 if float(config.ENV_DATA['ocs_version']) >= 4.6 else 1
 
     if config.ENV_DATA.get('platform') == constants.IBM_POWER_PLATFORM:
         min_eps = 1

